### PR TITLE
Fixed createStore

### PR DESCRIPTION
**Pull request by sajid-techis**

### CODE REVIEW

File: backend/config/settings.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 DEBUG = True
 
 ALLOWED_HOSTS = ['127.0.0.1', 'forum-prod-api.herokuapp.com']
-CORS_ALLOW_ALL_ORIGINS = False
```
Comment (kayato) [inline backend/config/settings.py:33]: Is this change needed?

File: frontend/src/reducks/store/store.js [hunk 2]
```diff
@@ -1,22 +1,26 @@
-import { applyMiddleware, combineReducers, configureStore } from 'redux';
```
Comment (kayato) [inline frontend/src/reducks/store/store.js:1]: What is the purpose of changing this file?

File: frontend/src/reducks/store/store.js [hunk 3]
```diff
@@ -1,22 +1,26 @@
...
-import { composeWithDevTools } from 'redux-devtools-extension';
-import { logger } from 'redux-logger';
-import thunk from 'redux-thunk';
+import {
+  applyMiddleware,
+  combineReducers,
+  legacycreateStore as configureStore,
+} from "redux";
```
Comment (kayato) [inline frontend/src/reducks/store/store.js]: We are using single quotations for other files, not double quotations. If you just formatted in your default settings, please just revert this change from your pull request.

File: backend/config/settings.py (hunk 1 above)
Comment (kayato) [inline backend/config/settings.py:33]: Do we need this change?

Comment (sajid-techis) [reply backend/config/settings.py:33]: Sir if we don't allow the cross-origins it gives error while deployment

File: frontend/src/reducks/store/store.js (hunk 2 above)
Comment (sajid-techis) [reply frontend/src/reducks/store/store.js:1]: Sir if we use CreateStore it gives us a warning to use configure store

File: frontend/src/reducks/store/store.js (hunk 3 above)
Comment (sajid-techis) [reply frontend/src/reducks/store/store.js]: Sure sit I will fix them.

File: backend/config/settings.py (hunk 1 above)
Comment (sajid-techis) [reply backend/config/settings.py:33]: We need to allow cross-origins if we don't it gives errors while deployment.

File: frontend/src/reducks/store/store.js (hunk 2 above)
Comment (sajid-techis) [reply frontend/src/reducks/store/store.js:1]: image

**Comment by ddeveza** [comment]
approved
